src/App.tsx

```diff
@@ -17,7 +21,24 @@ export const AuthContext = createContext<Partial<AuthContext>>({})
 type WebsocketContext = InstanceType<typeof Client> | null
 export const WebsocketContext = createContext<WebsocketContext>(null)
 
+// options for different color modes
+const modes = [
+  'light',
+  'dark',
+]
+
+// merge the color mode with the base theme
+// to create a new theme object
+const getTheme = mode =>
```

Comment:
If my suggestion, below, is followed, then the parameter passed into this function will be a boolean instead of a string, so a translation of `false` -> `'light`, `true` -> `'dark'` would need to take place here.